guide/principles/principles.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Bitcoin design principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 2
+has_children: false
+permalink: /guide/principles/principles/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system without intermediaries. To make this a reality, we encourage everyone that is working on products to deliberately support the core principles of designing for Bitcoin. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, all applications should strive to follow these principles. Diverging from them should only be done with very good reason.
```

Comment:
How about adding that these are principles that we have identified as a community? Not sure we should impose absolute views here.

Reply:
Added. 62d7a7a